### PR TITLE
in out fixes

### PR DESCRIPTION
**Pull request by TaerMosa**
in out fixes
1- linkDrawing 
2-DataBase

### CODE REVIEW

File: src/app/configuration/elementsFunctionality/computationalPart.ts [hunk 1]
```diff
@@ -1,66 +0,0 @@
...
-  // convert the elements from strings to numbers
-  const numbersArray = valuesArray.map(item => +item);
-  // sum the values of the array
-  return numbersArray.reduce((a, b) => a + b);
-}
-function subtract(valuesArray) {
-  // convert the elements from strings to numbers
-  const numbersArray = valuesArray.map(item => +item);
-  // subtract the right values of the array from the most left value
-  return numbersArray.reduce((a, b) => a - b);
```
Comment (NataliLevi) [inline src/app/configuration/elementsFunctionality/computationalPart.ts]: why did you delete the file?

Comment (TaerMosa) [reply src/app/configuration/elementsFunctionality/computationalPart.ts]: maybe by mistake just  merge the first code 
linkdrawing & database 

16354fc

File: src/app/configuration/elementsFunctionality/valueHandle.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+export const valueHandle = {
+
+    //  When a value of an object is updated, if a state exists - its value will be updates, otherwise - a new state will be added with the new value
+    updateState(cell, value) {
+        cell.set('previousValue', value);
+        var statesNumber = 0;   //currently only one value for object is allowed
+        _.each(cell.getEmbeddedCells(), function(child) {
+            statesNumber ++;
+            //There is already a state with the value set to the object - no need to update
```
Comment (NataliLevi) [inline src/app/configuration/elementsFunctionality/valueHandle.ts]: you must pull before making changes. this file is no longer exist and if I except your pull request, the "computationalPart" file will be deleted and this one will be added

**Comment by hanankohen** [comment]
Old code - already included in later puul
